engine/dlib/src/dlib/connection_pool.cpp

```diff
@@ -227,7 +227,7 @@ namespace dmConnectionPool
     static void PurgeExpired(HPool pool) {
         uint32_t n = pool->m_Connections.Size();
 
-        uint64_t now = dmTime::GetTime();
+        uint64_t now = dmTime::GetMonotonicTime();
```

Comment:
In all places where we need to measure time intervals, I replaced `dmTime::GetTime()` with  `dmTime::GetMonotonicTime()` to avoid unexpected issues